drag_and_drop_v2/public/css/drag_and_drop.css

```diff
@@ -666,11 +666,24 @@
     z-index: 1500;
 }
 
+.xblock--drag-and-drop .modal-dismiss-button {
+    font-size: 24px;
+    position: absolute;
+    top: 3px;
+    right: 3px;
```

Comment:
Have you verified this looks OK with rtl languages? I'm asking because only "right" is specified here.